server/boot/user.js

```diff
@@ -584,4 +582,14 @@ module.exports = function(app) {
       res.redirect('/map');
     }
   }
+
+  function flashCheater(req, user) {
```

Comment:
Ah of course. I was going to add the 'if user.isCheater' inside this function but decided against it.